### PR TITLE
add gdscript

### PR DESCRIPTION
**Pull request by tjdevries**
First pass at gdscript

### CODE REVIEW

**Comment by theHamsta** [comment]
Our CI is unhappy with you using `@function.call` (not documented). I guess we replaced that by just `@function`. Would really welcome `@function.call`

File: queries/gdscript/highlights.scm [hunk 1]
```diff
@@ -0,0 +1,27 @@
+(identifier) @variable
+
+(name) @variable
+(type) @type
+(comment) @comment
+
+(enum_definition
+  (name) @constant) @Statement
```
Comment (theHamsta) [inline queries/gdscript/highlights.scm:8]: We don't have `@Statement`

Comment (vigoux) [reply queries/gdscript/highlights.scm:8]: We actually do, it is a shortcut to set highlights directly from their groups, not recommended though...

Comment (theHamsta) [reply queries/gdscript/highlights.scm:8]: I was not trying to say that it won't work. It's just not in our `TSHighlights`. If there is a strong reason to use it that is very language specific I'd say it ok.

Comment (rockerBOO) [reply queries/gdscript/highlights.scm:8]: I think this would set `TSConstant` as well as the `Statement` groups.

**Comment by theHamsta** [comment]
Just rebased and fixed the CI for this PR.

**Review by vigoux — APPROVED** [review]

**Comment by vigoux** [comment]
This needs a rebase (again).

**Pull request merged**